README.md

```diff
@@ -1,655 +1,288 @@
-sysfonts::font_add_google('Abel','abel')
+* `6_visualize/out/river_conditions_[month start]_[month end]_[year]_twitter.mp4`
+* `6_visualize/out/river_conditions_[month start]_[month end]_[year]_visid.mp4`
+* `6_visualize/out/river_conditions_[month start]_[month end]_[year]_facebook.mp4`
+* `6_visualize/out/river_conditions_[month start]_[month end]_[year]_reddit.mp4`
+* `6_visualize/out/river_conditions_[month start]_[month end]_[year]_carousel.png`
+* `6_visualize/out/river_conditions_[month start]_[month end]_[year]_visid_thumbnail.png`
+* `6_visualize/out/river_conditions_[month start]_[month end]_[year]_square_thumbnail.png`
+* `6_visualize/out/river_conditions_[month start]_[month end]_[year]_insta.mp4`
 
-scipiper::scmake('6_intro_gif_tasks.yml', remake_file = '6_visualize.yml', force = TRUE)
-scipiper::scmake('6_visualize/log/6_intro_gif_tasks.ind', remake_file = '6_visualize.yml', force=TRUE)
+### Change the animation dates and download/process the new data
 
-scipiper::scmake('6_final_gif_tasks.yml', remake_file = '6_visualize.yml', force = TRUE)
-scipiper::scmake('6_visualize/log/6_final_gif_tasks.ind', remake_file = '6_visualize.yml', force=TRUE)
+You can do this step the day after the desired visualization date range ends. We typically build these quarterly, so you would kick this step off on January 1 for quarter 1's animation (October 1 - December 31), and so on for the other quarters. Typically, we start this step on ...
 
-scipiper::scmake('6_timestep_gif_tasks.yml', remake_file = '6_visualize.yml', force = TRUE)
-scipiper::scmake('6_visualize/log/6_timestep_gif_tasks.ind', remake_file = '6_visualize.yml', force=TRUE)
+* January 1st for quarter one's (Oct 1 - Dec 31) animation 
+* April 1st for quarter two's (Jan 1 - Mar 31) animation 
+* July 1st for quarter three's (Apr 1 - Jun 30) animation - *note that this timing can be tricky given the July 4th holiday*
+* October 1st for quarter four's (Jul 1 - Sep 30) animation 
 
-scipiper::scmake('6_pause_gif_tasks.yml', remake_file = '6_visualize.yml', force=TRUE)
-scipiper::scmake('6_visualize/log/6_pause_gif_tasks.ind', remake_file = '6_visualize.yml', force=TRUE)
+#### Update the dates
 
-scipiper::scmake('6_visualize/out/year_in_review.mp4', remake_file = '6_visualize.yml', force = TRUE)
+Go to `viz_config.yml` and make sure that the start/end entries for both `vizDates` and `wheelDates` are appropriate. `vizDates` refer to the dates that will be shown as animation frames and for which data will be pulled. `wheelDates` should be the start/end dates for the current water year and are used to construct the date wheel visual on the bottom left of the animation.
 
-```
+Next, scroll down in `viz_config.yml` and ensure that the field `title_cfg > subtitle` shows the appropriate dates. While you're at it, update the same field but in the commented out area which will be used later to create an Instagram version of the animation.
```

Comment:
Yes, but would require altering some of the functions inputs/outputs. Since it is so nested, I end up just changing the text here instead of updating the function.

Reply:
^ Could make for a good issue/improvement